src/App.js

```diff
@@ -88,9 +88,17 @@ class App extends Component {
           </Helmet>
           <OriginCatcher />
           <Switch>
-            <Route path="/" exact>
-              <RedirectToLegacyUrl to="/Campaigns/Draft" />
-            </Route>
+            <Route
+              exact
+              path="/"
+              render={({ location }) =>
+                location.hash.length ? (
+                  <Redirect to={location.hash.replace('#', '')} />
```

Comment:
What do you think about removing the first character in place of replacing all `#`?

All `hash` values start with `#`, right? So, it is cheaper and more general because it will keep other `#` occurrences (if there are more than one)

Reply:
Does it keep the page in history or skip it?

I do not know what option is better, but we should test it to ensure good user experience.

Reply:
It works locally and that's all, it needs further testing. I should check history too..

Reply:
History seems to be working great! 😄

Reply:
> What do you think about removing the first character in place of replacing all `#`?
> 
> All `hash` values start with `#`, right? So, it is cheaper and more general because it will keep other `#` occurrences (if there are more than one)

Only gets to `/` when the hash is in the route like this `http://localhost:3000/#/integrations/shopify` but if we have `http://localhost:3000/integrations/shopify#hash1` it never gets to `/` rule.
If whe have `http://localhost:3000/#/integrations/shopify#hash1` seems to only detect as hash the first hash and then redirects to `http://localhost:3000/integrations/shopify#hash1` without issues.

Reply:
Ohh, _replace_ replace in javascript only takes the first occurrence, right?

Reply:
If we were to test in cdn, redirects are working in a very weird fashion, `http://cdn.fromdoppler.com/doppler-webapp/build1606/#/login` redirects to `https://cdn.fromdoppler.com/reports` O_o

Reply:
It is because `/reports` is a kind of default page.

Reply:
Anyway, testing it in the CDN has no sense because we need changes in the HTTP server. But it generates a new issue for us: what will we do with our CDN after this change?

Reply:
Seems to be replacing pathname with location.hash, in local is the same, but in cdn url is not the same.

Reply:
Tiene que ser porque localmente el root path es `/`, pero en el CDN es algo como `/doppler-webapp/build76/`.